CODEOWNERS

```diff
@@ -155,7 +157,7 @@
 /gallery/laws-insights/** @microsoft/loganalytics
 /gallery/updatecenter-insights/** @microsoft/azure-update-center
 /gallery/workbook/Azure*Update*Manage*Center.json @microsoft/azure-update-center
-/gallery/workbook/microsoft.operationalinsights*workspaces.json @microsoft/loganalytics
+/gallery/workbook/microsoft.operationalinsights*workspaces.json @microsoft/loganalytics @microsoft/serviceshub
```

Comment:
do you really want this one?  in this part you're now claiming that you *own* this gallery and can review+approve any changes to this gallery?

Reply:
I assumed it was part of taking responsibility to the changes I made there. Do you suggest otherwise?